app/screens/channel_members/index.js

```diff
@@ -30,7 +30,7 @@ function makeMapStateToProps() {
             requestStatus: state.requests.users.getProfilesInChannel.status,
             searchRequestStatus: state.requests.users.searchProfiles.status,
             removeMembersStatus: state.requests.channels.removeChannelMember.status,
-            canManageUsers: canManageChannelMembers(state),
+            canManageUsers: canManageChannelMembers(state) && currentChannel.delete_at === 0,
```

Comment:
AFAIK (We can ask @wiersgallak here) You can still manage the users in archived channels if you have the permissions to manage the users. The users of an archived channel is not "locked". So, I would say that the interface should allow you to manage the users if you still can. Please @wiersgallak confirm that.

Reply:
Ah OK. I assumed it was locked. Good callout. @wiersgallak I'll await your direction.

Reply:
Yes @mkraft - I agree that we will still want the ability to manage users in archived channels as there are use cases that support the necessity to do so.